ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -404,7 +404,7 @@ def line_is_expected(self, str, expect_messages_regex):
         '''
 
         ret_code = False
-        if (expect_messages_regex is not None) and (expect_messages_regex.findall(str)):
+        if (expect_messages_regex is not None) and (expect_messages_regex.match(str)):
```

Comment:
Limit this change to advanced-reboot cases to avoid blast radius if there is an impact to other cases.